de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -101,6 +101,66 @@ Now press the link btn:[Workbench].
 
 Done. You installed and successfully started Virtual Satellite Developer IDE.
 
+==== IDE Git setup
+
+Once you have installed your Virtual Satellite Developer IDE, you need to set up your Git IDE. 
+Following the next steps will help you with the progress.
+
+
+[%header,cols=">8,62a,40a"] 
+|===
+
+|Step
+|Task
+|Image
+
+|{counter:steps}.
+|Start with opening a new perspective by clicking the "window" button in your menu. +
+Select the option "Git". 
+|image::chapterIdeInstallation/subsectionGitIdeSetup/2_OpenPerspective.png[Open perspective]
+
+|{counter:steps}.
+|In the next step you have to select a repository source.
+Choose "clone URl" and continue. +
+After that you need to visit www.github.com and choose the repository you want to clone. +

```

Comment:
Since this is a VirSat IDE setup tutorial maybe we could link directly to the virtual satellite organization page here https://github.com/virtualsatellite

Reply:
I Agree.

Reply:
I changed the link.